docs/resources/Guild_Event.md

```diff
@@ -0,0 +1,100 @@
+| -------------------- | ------------------------------------------------------------------------------------- | --------------------------------- |
+| channel_id?          | snowflake                                                                             | the stage channel id of the event |
+| name                 | string                                                                                | the name of the event             |
+| privacy_level        | [privacy level](#DOCS_RESOURCES_STAGE_INSTANCE/stage-instance-object-privacy-level)   | the privacy level of the event    |
+| scheduled_start_time | ISO8601                                                                               | the time to schedule the event    |
+| description?         | string                                                                                | the description of the event      |
+| entity_type?         | [entity type](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object-guild-event-entity-type) | event entity type                 |
+
+## List Guild Events % GET /guild-events
+
+Returns a list of [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) objects available to the bot.
+
+## Get Guild Event % GET /guild-events/{event.id#DOCS_RESOURCES_GUILD_EVENT/guild-event-object}
+
+Get a guild event. Returns a [Guild Event](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object) object on success.
+
+## Delete Guild Event % DELETE /guild-events/{event.id#DOCS_RESOURCES_GUILD_EVENT/guild-event-object}
+
+Delete a guild event.
+
+## Modify Guild Event % PATCH /guild-events/{event.id#DOCS_RESOURCES_GUILD_EVENT/guild-event-object}
+
+Modify a guild event.
+
+###### JSON Params
+
+| Field                | Type                                                                                  | Description                       |
+| -------------------- | ------------------------------------------------------------------------------------- | --------------------------------- |
+| channel_id?          | snowflake                                                                             | the stage channel id of the event |
+| name                 | string                                                                                | the name of the event             |
+| privacy_level        | [privacy level](#DOCS_RESOURCES_STAGE_INSTANCE/stage-instance-object-privacy-level)   | the privacy level of the event    |
+| scheduled_start_time | ISO8601                                                                               | the time to schedule the event    |
+| description?         | string                                                                                | the description of the event      |
+| entity_type?         | [entity type](#DOCS_RESOURCES_GUILD_EVENT/guild-event-object-guild-event-entity-type) | event entity type                 |
```

Comment:
Is it intended that most of these parameters are marked as not optional?

Reply:
You don't need to edit all params at once. If anything they should all be optional imo

Reply:
since it's a PATCH endpoint, ti should have the standard "all params are optional" info box

Reply:
I opted for adding the info box and leaving out any `?` at the end of the names.